lib/src/cmaf/README.md

```diff
@@ -0,0 +1,56 @@
+# CMAF HAM 
+
+## Overview 
+
+HLS and DASH stand as the predominant video streaming technologies currently. Consequently, users often encounter challenges such as converting between HLS and DASH, manipulating manifests, and programmatically understanding manifest structures.
+
+The Common Media Application Format (CMAF) for segmented media (ISO/IEC 23000-19) addresses these challenges by defining a universal format based on ISOBMFF. Additionally, it introduces the Hypothetical Application Model, a framework illustrating the practical usage of CMAF segments and fragments in streaming applications. This project is inspired by the principles outlined in the CMAF standard and the [Hypothetical Application Model] ([Hypothetical Application Model](https://cdn.cta.tech/cta/media/media/resources/standards/cta-5005-a-final.pdf)).
+
+## Features 
+
+* Fundamental on-demand conversion between [HLS] and [DASH] manifests, covering transformations such as HLS to HAM, HAM to HLS, DASH to HAM, and HAM to DASH.
+* Elementary Media Presentation querying functionality.
+
+## Usage 
+
+Here's a sample demonstration illustrating how to utilize the features of the CMAF HAM module in TypeScript:
+
+
+```typescript
+import {
+  hamToHls,
+  hamToDash,
+  hlsToHam,
+  dashToHam,
+} from "@svta/common-media-library";
+
+
+// Parse from DASH format
+const parseFromDash = (input: string) => {
+  const hamObj = mpdToHam(input); // Convert input to HAM object
+  const hlsManifest = hamToHls(hamObj); // Convert HAM object to HLS manifest
+  const dashManifest = hamToDash(hamObj); // Convert HAM object to DASH manifest
+};
+
+// Parse from HLS format
+const parseFromHls = (main: string, playlists: string[]) => {
+  const hamObj = hlsToHam(main, playlists); // Convert HLS to HAM object
+  const hlsManifest = hamToHls(hamObj); // Convert HAM object to HLS manifest
+  const dashManifest = hamToDash(hamObj); // Convert HAM object to DASH manifest
+};

```

Comment:
Same as previous comment, remove the arrow function (it's weird) and define the variables with complete names.

Probably creating a new code block to define the hls functions, so the name of the variables can be repeated

Reply:
I replaced arrow functions, let me know if it makes sense for you the way it is right now